resources/pool.rb

```diff
@@ -80,7 +80,7 @@
 property :smp_processor_affinity_mask, Float, default: 4_294_967_295.0, coerce: proc { |v| v.to_f }
 property :smp_processor_affinity_mask_2, Float, default: 4_294_967_295.0, coerce: proc { |v| v.to_f }
 
-property :running, [true, false], desired_state: true
+property :running, [true, false]
```

Comment:
@EasyAsABC123 My reading of https://docs.chef.io/custom_resources.html#desired-state is that `desired_state: true` means that the property will get its value from the current state of the system (and not via the instance of the resource in the recipe).